ansible_base/api_documentation/apps.py

```diff
@@ -13,3 +13,10 @@ def ready(self):
 
         if 'ansible_base.authentication' in settings.INSTALLED_APPS:
             apply_authentication_customizations()
+
+        # Import filter extensions to register them with drf-spectacular
+        try:
+            from ansible_base.api_documentation import filter_extensions  # noqa: F401
+        except ImportError:
+            # drf_spectacular may not be available
+            pass
```

Comment:
`ansible_base.rest_filters` is also an optional django application that a service /could/ use. We might want to wrap this in something like:

```suggestion
        if 'ansible_base.rest_filters' in settings.INSTALLED_APPS and 'ansible_base.api_documentation' in settings.INSTALLED_APPS:
            # If this service is using DAB rest filters and api documentation, load our filter extensions for OpenAPI
            from ansible_base.api_documentation import filter_extensions  # noqa: F401
```